### ISSUE TITLE
two-handed buckler mounted

### ISSUE THREAD

**Issue by vinince** [ScaredKurufinve/Way-of-the-shield#28]
Hi SK.

I've noticed two issues with using buckler with two-handed weapons while mounted.
One is that the **-1 attack** penalty is gone when you mount. And the other is that you can't re-equip a buckler with your two-handed weapon while mounted, it just replaces the weapon like vanilla, but it stays there if you are equiped with it before mounting.

**Comment by ScaredKurufinve** [comment]
Hm, indeed.
I did fix the problem with being unable to re-equip buckler while mounted. For now posted as pre-release here https://github.com/ScaredKurufinve/Way-of-the-shield/releases/tag/v2.0.10
I'll see tomorrow to why there's no penalty.

**Comment by vinince** [comment]
Nice, thanks! And I believe you just fixed both issues. I can now re-equip the buckler on 2-handed wep while mounted and I can also see the correct penalty.

**Comment by ScaredKurufinve** [comment]
Interesting, because I don't see the penalty :D

**Comment by vinince** [comment]
Really? That's weird. Maybe your character is using Unhindering Shield feat?

![image](https://github.com/ScaredKurufinve/Way-of-the-shield/assets/41359354/46043c13-6a21-44d6-86dc-4db50cafbf1b)
Tried with more than one character, mounted or on foot, looks all good here.

**Comment by ScaredKurufinve** [comment]
> Maybe your character is using Unhindering Shield feat?
No, that's because my character has Use Weapon One-handedly active, and it affects the weapon even though it shouldn't :/

**Comment by vinince** [comment]
> > Maybe your character is using Unhindering Shield feat?
> 
> No, that's because my character has Use Weapon One-handedly active, and it affects the weapon even though it shouldn't :/

Hmm I tried activating that but for me it's affecting only one-handed weapons like longswords and such, as it should be. Maybe something is different on my settings, or some other mod that I'm using, idk.


Also I just noticed that the **Spiked Masterwork Heavy Shield** is giving me that wrong _-4 Not proficient with this weapon_ penalty while using Shield Bash. A +1 common heavy shield is fine but not that one. Weird since both are Heavy Shield category...

**Comment by ScaredKurufinve** [comment]
SpikedHeavyShield is an entirely separate category from HeavyShield. Somehow I thought it's not used for actual items in the game despite the name.

**Comment by vinince** [comment]
lmao, gotta be influence of another mod I'm using along, cause this last version really messed up things badly for me in more than one way haha! But i'ts all good, since the previous one was doing perfect here, I'll just stick with it, no need to trouble you no more. My thanks!

**Comment by vinince** [comment]
Well my character was holding a Glaive with a Buckler with no penalty (even on foot). Then I tried the same equipment on Seelah and she got penalties like if she was two-weapon fighting with a Glaive and a Buckler lol.
Tried hiting the One Handed on and off but no difference.

But again, I'm using the mod along this many other, so maybe that's the cause.
![image](https://github.com/ScaredKurufinve/Way-of-the-shield/assets/41359354/e46a9bbd-4118-42e3-80b1-75af9c3fbaa6)

**Comment by ScaredKurufinve** [comment]
this is kinda strange 🤔 Can it be that your Seelah has Buckler Parry activated? It and Shielded Defense are the only 2 abilities in my mod which may force TWF penalties on a character.

**Comment by vinince** [comment]
Both options are always deactivated here, I don't use any homebrews, tabletop only.
I think it's better to wait for someone else report any issues, to see if it's only on my end.

And since I'm not using the spike shield, that previous version is perfect already for my needs. 👍
